Makefile

```diff
@@ -70,4 +70,7 @@ integ-tests-with-docker-old:
 	make ARCH=old compile-with-docker
 	make prep-python
 	make TEST_ARCH="" TEST_PORT=9052 exec-python-e2e-test
-	
\ No newline at end of file
+
+check-binaries: prep-python
```

Comment:
So with this, we could run the make command or the workflow to check for vulnerabilities? If so, do we need both?

Reply:
I'm not sure I fully understand the question, so let me know if this answers it. Now it will be possible to manually do `make check-binaries` in your local environment if you want to check locally without waiting for the GitHub action to run.

Reply:
Yeah that's what I meant